src/include/platform/CHIPDeviceConfig.h

```diff
@@ -392,27 +392,67 @@
 #endif
 
 /**
- * CHIP_DEVICE_CONFIG_BLE_FAST_ADVERTISING_INTERVAL
```

Comment:
Btw, don't you need to update platforms which use the old constants?

Reply:
Yeah, I realized that after all builds failed :)